### PR TITLE
Populate world

### PR DESCRIPTION
**Pull request by beta-version-profile**
### js-pre-oop

[Demo](https://beta-version-profile.github.io/a-tiny-JS-world) |
[Code base](https://github.com/beta-version-profile/a-tiny-JS-world/blob/gh-pages/index.js)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by AMashoshyna** [comment]
Great job!

My comments:
- functions and methods are usually called with verbs in initial form, so: toSay => say
- commit messages should be more informative (yes, we all call our commits "fix" sometimes, but while you learn try to pay more attention to such things).

Otherwise LGTM

**Comment by zonzujiro** [comment]
You have a lot of changes, from other tasks too. Is it ok?

File: submissions/beta-version-profile/a-tiny-JS-world_/index.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  }
+}
+class Cat extends Alfa {
+    constructor(species, name, gender, legs, hands) {
+      super(species, name, gender, legs, hands)
+      this.saying = 'meow - meow'
+    }
+}
+
+const catwoman = new Cat('human', 'Jenny', 'woman', 2, 2, 'Hello, world!!!')
```
Comment (OleksiyRudenko) [inline submissions/beta-version-profile/a-tiny-JS-world_/index.js]: Looks like the last argument passed to the constructor gets ignored.
Please, get rid of obsolete code.

File: submissions/beta-version-profile/a-tiny-JS-world_/index.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  constructor(species, name, gender, legs, hands, saying) {
+    this.species = species
+    this.legs = legs
+    this.hands = hands
+    this.name = name
+    this.gender = gender
+    this.saying = saying
+  }
+  say(){ 
+    print(`species: <strong>${this.species}</strong>; name: <strong>${this.name}</strong>; gender: <strong>${this.gender}</strong>; legs: <strong>${this.legs}</strong>; hands: <strong>${this.hands}</strong>; saying: <strong>${this.saying}</strong>;`
```
Comment (OleksiyRudenko) [inline submissions/beta-version-profile/a-tiny-JS-world_/index.js]: Not related to OOP. Too many repetitions. Try making this part DRYer.
You may find the following methods helpful: `#Object.keys()`, `#Array.map()` and `#Array.join()`

File: submissions/beta-version-profile/a-tiny-JS-world_/index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    )
+  }
+}
+class Cat extends Alfa {
```
Comment (OleksiyRudenko) [inline submissions/beta-version-profile/a-tiny-JS-world_/index.js]: This technically works however the code reads that a Cat is like an evolutionary step from Alfas. Why not vice versa? Both approaches are questionable.
You are NOT required to fix this now. Pay special attention to inheritance and composition vs inheritance topics when studying OOP modules that follow.

**Comment by OleksiyRudenko** [comment]
@beta-version-profile you have resolved the issues rewriting the history forcedly. It is OK here unless it has broken your other task history. Please, try avoiding rewriting history that is already a part of any repo remotes.

File: submissions/beta-version-profile/a-tiny-JS-world_/index.js (hunk 3 above)
Comment (OleksiyRudenko) [reply submissions/beta-version-profile/a-tiny-JS-world_/index.js]: Much better! Thanks!

**Comment by zonzujiro** [comment]
@OleksiyRudenko oi! Pls, take a look when you will have a free time

**Pull request merged**